src/Service/Nph/NphOrderService.php

```diff
@@ -399,6 +397,21 @@ private function generateOrderSummaryArray(array $nphOrder): array
         return $returnArray;
     }
 
+    public function hasAtLeastOneAliquotSample(array $formData, string $sampleCode): bool
+    {
+        $aliquots = $this->getAliquots($sampleCode);
+        foreach (array_keys($aliquots) as $aliquotCode) {
+            if (isset($formData[$aliquotCode])) {
+                foreach ($formData[$aliquotCode] as $aliquotId) {
+                    if ($aliquotId) {
+                        return true;
+                    }
+                }
+            }
```

Comment:
Could this be simplified to: 
```suggestion
            if (isset($formData[$aliquotCode]) && count($formData[$aliquoteCode]) > 0) {
                return true;
            }
```

Reply:
@thornview This is how the collection form data is represented. The count will be always > 0, so we need iterate the loop and determine if the field has null value or not.

![image](https://user-images.githubusercontent.com/6041980/206478890-ec07a4ee-134b-414f-9b2e-9564f7a6e9c5.png)